src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,191 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.pagefactory.AjaxElementLocatorFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class BasePage {
+
+//    private String isSuccessLocator =".success.show";
+//    private String playButtonAreaLocator = "[class='album-thumb']";
+//    private String playButtonLocator = "[data-testid='play-btn']";
+//    private String pauseButtonLocator = "[data-testid='pause-btn']";
+//    private String linkAllSongsLocator = "[class='songs']";
+
+
+
+    @FindBy(css=".success.show")
+    private WebElement isSuccessElement;
+    @FindBy(css= "[class='album-thumb']")
+    private WebElement playButtonAreaElement;
+    @FindBy(css= "[data-testid='play-btn']")
+    private WebElement playButtonElement;
+    @FindBy(css= "[data-testid='pause-btn']")
+    private WebElement pauseButtonElement;
+    @FindBy(css="[class='songs']")
+    private WebElement linkAllSongsElement;
```

Comment:
Base Page \ Base Class — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The methods elements must be stored in the `class HomePage` because the elements are visible on the web site -home page


![Screenshot_1](https://user-images.githubusercontent.com/104761482/230761708-a9f0e7e0-2a2b-45f4-aa0c-76d4194926aa.jpg)

Reply:
Hi, ok, i've moved it